kolibri/core/assets/src/api-resources/exam.js

```diff
@@ -7,6 +7,15 @@ class ExamResource extends Resource {
   static idKey() {
     return 'id';
   }
+
+  /**
+   * Returns a Collection of Exams filtered by a collectionId
+   * @param {string} collectionId - e.g. a classId
+   * @returns {Collection<Exam>}
+   */
+  getCollectionForClass(collectionId) {
```

Comment:
Given that it only gets used once, and that it is a common pattern to filter by collection getParams, I think this is an over abstraction.